### PR TITLE
OCPBUGS#18100-New: Revert optional step in vSphere docs

### PR DESCRIPTION
**Pull request by dfitzmau**
[OCPBUGS-18100](https://issues.redhat.com/browse/OCPBUGS-18100)

Version(s):
4.14 through to 4.10

Link to docs preview:
* [Installing RHCOS and starting the OpenShift Container Platform bootstrap process](https://64010--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere#installation-vsphere-machines_installing-vsphere)
* [Adding more compute machines to a cluster in vSphere](https://64010--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere#machine-vsphere-machines_installing-vsphere)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/installation-vsphere-machines.adoc [hunk 1]
```diff
@@ -141,10 +141,9 @@ Ensure that all virtual machine names across a vSphere installation are unique.
 
 .. On the *Select a compute resource* tab, select the name of a host in your datacenter.
 +
-.. On the *Select clone options*, select
-*Customize this virtual machine's hardware*.
+.. On the *Select clone options*, select *Customize this virtual machine's hardware*.
```
[the remark this answers is not in the document]
Comment (dfitzmau) [reply modules/installation-vsphere-machines.adoc:144]: Screenshot from 2023-08-29 11-58-41

[the remark this answers is not in the document]
Comment (sgaoshang) [reply modules/installation-vsphere-machines.adoc:144]: yes, I think so

**Comment by sgaoshang** [comment]
Compared with 4.9 [doc](https://docs.openshift.com/container-platform/4.9/installing/installing_vsphere/installing-vsphere.html#installation-vsphere-machines_installing-vsphere), I have some questions, pls see bellow

image

**Comment by sgaoshang** [comment]
And I'm a little confused why we have this line added
image

**Comment by dfitzmau** [comment]
Hi @sgaoshang . Thanks for the review. Latency sensitivity information was removed because of https://issues.redhat.com/browse/OSDOCS-4561. Here is the exact location in the PR where the content was removed: [Removed latency sensitivity](https://github.com/openshift/openshift-docs/pull/61358/files#:~:text=Optional%3A%20In%20the%20event%20of%20cluster%20performance%20issues%2C%20from%20the%20*Latency%20Sensitivity*%20list%2C%20select%20*High*.%20Ensure%20that%20your%20VM%27s%20CPU%20and)

File: modules/installation-vsphere-machines.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (dfitzmau) [reply modules/installation-vsphere-machines.adoc:144]: Thanks. I'll ensure I add a basic step in the following sections:

* Installing RHCOS and starting the OpenShift Container Platform bootstrap process
* Adding more compute machines to a cluster in vSphere

**Comment by dfitzmau** [comment]
I see that the 4.9 OVA statement was added in this [PR](https://github.com/openshift/openshift-docs/commit/7a1094ca20a5bc3e04df27f3f2a7da260cb07e5e)

**Comment by sgaoshang** [comment]
@dfitzmau Thanks so much for the PR pointed out, based on my manual testing process recently, here's my suggestion.
image
In Point 1, we need to add `Set the guestinfo.afterburn.initrd.network-kargs property before booting a VM from an OVA in vSphere`, only  `export IPCFG` won't work
image
In Point 2, we need to add `guestinfo.ignition.config.data` and `guestinfo.ignition.config.data.encoding` set, it can not be ommited.
image

**Comment by dfitzmau** [comment]
Thanks, @sgaoshang , for the screenshot. I updated the `Installing RHCOS and starting the OpenShift Container Platform bootstrap process` section. Please let me know your thoughts.

File: modules/machine-vsphere-machines.adoc [hunk 2]
```diff
@@ -39,14 +39,16 @@ Ensure that all virtual machine names across a vSphere installation are unique.
...
 .. On the *Select a compute resource* tab, select the name of a host in your datacenter.
+.. On the *Select storage* tab, select storage for your configuration and disk files.
 .. On the *Select clone options*, select *Customize this virtual machine's hardware*.
-.. On the *Customize hardware* tab, click *VM Options* -> *Advanced*.
-*** Click *Edit Configuration*, and on the *Configuration Parameters* window, click *Add Configuration Params*. Define the following parameter names and values:
+.. On the *Customize hardware* tab, click *Advanced Parameters*.
+*** Add the following configuration parameter names and values by specifying data in the *Attribute* and *Values* fields. Ensure that you select the *Add* button for each parameter that you create.
 **** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded compute Ignition config file for this machine type.
 **** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
 **** `disk.EnableUUID`: Specify `TRUE`.
```
Comment (jeana-redhat) [inline modules/machine-vsphere-machines.adoc:48]: Wondering what you think about this as an alternative to reduce the nesting a bit:
```suggestion
.. Add the following configuration parameter names and values by specifying data in the *Attribute* and *Values* fields. Ensure that you select the *Add* button for each parameter that you create.
*** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded compute Ignition config file for this machine type.
*** `guestinfo.ignition.config.data.encoding`: Specify `base64`.
*** `disk.EnableUUID`: Specify `TRUE`.
```

File: modules/machine-vsphere-machines.adoc [hunk 3]
```diff
@@ -39,14 +39,16 @@ Ensure that all virtual machine names across a vSphere installation are unique.
...
-.. In the *Virtual Hardware* panel of the *Customize hardware* tab, modify the specified values as required. Ensure that the amount of RAM, CPU, and disk storage meets the minimum requirements for the machine type. Also, make sure to select the correct network under *Add network adapter* if there are multiple networks available.
-.. Complete the configuration and power on the VM.
+.. In the *Virtual Hardware* panel of the *Customize hardware* tab, modify the specified values as required. Ensure that the amount of RAM, CPU, and disk storage meets the minimum requirements for the machine type. If many networks exists, select *Add New Device* > *Network Adapter*, and then enter your network information in the fields provided by the *New Network* menu item.
```
Comment (jeana-redhat) [inline modules/machine-vsphere-machines.adoc:49]: Maybe break this up a bit for reading ease?
```suggestion
.. In the *Virtual Hardware* panel of the *Customize hardware* tab, modify the specified values as required. Ensure that the amount of RAM, CPU, and disk storage meets the minimum requirements for the machine type.
+
If many networks exists, select *Add New Device* > *Network Adapter*, and then enter your network information in the fields provided by the *New Network* menu item.
```

File: modules/machine-vsphere-machines.adoc
Comment (jeana-redhat) [inline modules/machine-vsphere-machines.adoc:1]: Out of scope for this PR, but food for thought: 

This is a two-step procedure, and the first step is 10 substeps. Would it make sense to pull the text of step 1 into the intro material, and then make step 2 a singleton under a `.Next steps` heading?

File: modules/installation-vsphere-machines.adoc [hunk 4]
```diff
@@ -141,20 +141,20 @@ Ensure that all virtual machine names across a vSphere installation are unique.
...
```
Comment (jeana-redhat) [inline modules/installation-vsphere-machines.adoc:144]: `On the *Select clone options*`

Not sure if `the` should be removed or a noun (`tab`?) should be added here

**Review by jeana-redhat — COMMENTED** [review]
Only [one](https://github.com/openshift/openshift-docs/pull/64010#discussion_r1316110766) actual fix, the rest are suggestions (some of which are definitely out of scope for this PR but I thought worth a mention). 

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/installation-vsphere-machines.adoc (hunk 4 above)
Comment (dfitzmau) [reply modules/installation-vsphere-machines.adoc:144]: I went with "menu item" as I'm not sure if they are tabs. To me, "Advanced Parameters" is a tab.

File: modules/machine-vsphere-machines.adoc
Comment (dfitzmau) [reply modules/machine-vsphere-machines.adoc:1]: Good points, and thanks for the review. I incorporated all your feedback, including the file restructuring.

**Comment by jeana-redhat** [comment]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by jeana-redhat** [comment]
@dfitzmau hey, sorry this made a bunch of messages because I tired to CP before merge :sweat_smile: Anyway, 4.13+ worked, but 4.12 and below need manual picks.

**Pull request merged**